kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Constants.java

```diff
@@ -116,15 +94,16 @@ private Constants() {
      */
     public static final String DEPLOYMENT = "Deployment";
     public static final String DEPLOYMENT_TYPE = "deployment-type";
-    public static final String CUSTOM_RESOURCE_DEFINITION = "CustomResourceDefinition";
     public static final String CLUSTER_ROLE = "ClusterRole";
     public static final String CONFIG_MAP = "ConfigMap";
     public static final String JOB = "Job";
     public static final String NAMESPACE = "Namespace";
-    public static final String POD = "Pod";
     public static final String SECRET = "Secret";
     public static final String SERVICE = "Service";
     public static final String SERVICE_ACCOUNT = "ServiceAccount";
+    public static final String KAFKA_KIND = "Kafka";
```

Comment:
We should probably make these constants be prefixed with `STRIMZI_`.  As we introduce our own CR names into this code, things will become confusing otherwise.